modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/AppContext.es.js

```diff
@@ -18,12 +18,15 @@ const AppContext = React.createContext();
 const AppContextProvider = ({children, ...props}) => {
 	const [reindexStatuses, setReindexStatuses] = useState([]);
 	const [title, setTitle] = useState(Liferay.Language.get('metrics'));
+	const [shouldCalculateMetrics, setShouldCalculateMetrics] = useState(false);
```

Comment:
Hey @jotascript, Can you please rename to `setFetchDateModified` and `fetchDateModified`? Please propagate this changes to other places as well.

Thx